package ch04.ex01;

public class ExceptionTest03 {
	 public static void main(String[] args) {
		
		 System.out.println("매개변수 2개 전달");
		 
		 int a = 9;
		 int b = 3;
		 System.out.println("a=>" + a + ", b=>" + b);
		 System.out.println("a를 b로 나눈 몫=> ");
		 
	}
}
